scripts/pre_commit_linter.py

```diff
@@ -2423,8 +2423,8 @@ def _check_extra_js_files(self):
 
     def _check_constants_declaration(self):
         """Checks the declaration of constants in the TS files to ensure that
-        the constants are not declared in files other than *.constants.ts and
-        that the constants are declared only single time.
+        the constants are not declared in files other than *.constants.ajs.ts
+        and *.constants.ts that the constants are declared only single time.
```

Comment:
I don't see `*.constants.ts` being checked below, looks like you're missing that condition.

Reply:
I have removed the check for `*.constants.ts` since now all the `oppia.constant(` statements are in `*.constants.ajs.ts`.